### PR TITLE
V17.1.0 - In Progress

### PR DESCRIPTION
**Pull request by adlair**

### CODE REVIEW

File: firebase-firestore/nuget/Xamarin.Firebase.Firestore.template.nuspec [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <id>Xamarin.Firebase.Firestore</id>
     <title>Xamarin Firebase - Firestore</title>
     <version>$version$</version>
-    <authors>Microsoft</authors>
+    <authors>Xamarin Inc.</authors>
```
Comment (Redth) [inline firebase-firestore/nuget/Xamarin.Firebase.Firestore.template.nuspec]: This needs to change back to Microsoft.

File: firebase-firestore/source/Additions/Additions.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Android.OS;
+using Android.Runtime;
+using Java.Lang;
+
+namespace Com.Google.Firebase
```
Comment (Redth) [inline firebase-firestore/source/Additions/Additions.cs]: This namespace needs adjusting in metadata.xml

**Comment by JimmyRobz** [comment]
Why does it take that long to upgrade to new versions of Google Play Services and Firebase? It's a shame we can't use new APIs like ML Kit from a NuGet package... 

Can I use this PR in a real project? How? 

Thanks

**Comment by Redth** [comment]
We switched midstream to a different project build model which includes templated binding project generation to deal with dependency chains a bit more cleanly.

If you look at the 15.0.1-binderate branch you should be able to in theory build that and use it from source.  We will be evaluating updating the versions of things we bind on that branch VERY soon.

For now, closing this PR in favor of the branch mentioned.

**Pull request closed without merging**